web/.eslintrc

```diff
@@ -82,6 +82,12 @@
       }
     ],
     "no-multi-spaces": ["error", { "ignoreEOLComments": true }],
+    "react/no-unknown-property": [
```

Comment:
Is this for debug use? Don't commit it then

Reply:
It is for ignore css property in theme component. After update package.json, yarn build seem not work without   
this eslint config.